src/Tools/Source/RunTests/ProcessTestExecutor.cs

```diff
@@ -87,11 +87,16 @@ void MaybeAddSeparator(char separator = '|')
                 builder.Append(" --blame-crash");
             }
 
-            // The 25 minute timeout accounts for the fact that VSIX deployment and/or experimental hive reset and
+            // The 25 minute timeout in integration tests accounts for the fact that VSIX deployment and/or experimental hive reset and
             // configuration can take significant time (seems to vary from ~10 seconds to ~15 minutes), and the blame
```

Comment:
When we originally made this change we later discovered we were having other issues that was causing VSIX deployment to hang -- it'd be interesting to actually see if we actually can move it back to 15 minutes for everything.